lib/chef/knife/ec2_server_create.rb

```diff
@@ -67,10 +67,18 @@ class Ec2ServerCreate < Knife
         :proc => Proc.new { |groups| groups.split(',') }
 
       option :security_group_ids,
-        :short => "-g 'X,Y,Z'",
-        :long => "--security-group-ids 'X,Y,Z'",
-        :description => "The security group ids for this server; required when using VPC,Please provide values in format --security-group-ids 'X,Y,Z'",
-        :proc => Proc.new { |security_group_ids| security_group_ids.split(',') }
+        :short => "-g SECURITY_GROUP_IDS",
+        :long => "--security-group-ids",
```

Comment:
I have a feeling we should change this to `--security-group-id` which probably means having a completely separate `--security-group-ids` option that raises an error and tells you to use the new flag and that you should specify it multiple times instead of using a comma now, then we could take `--security-group-ids` out in a year or so.